src/routes/cities/[slug].svelte

```diff
@@ -18,26 +19,44 @@
 </script>
 
 <script lang="ts">
-	import { distanceBetweenCoordinates, formatTitle } from '../../utils'
+	import { distanceBetweenCoordinates, formatTitle } from '../../utils';
```

Comment:
oh can you enable [tabs to spaces](https://stackoverflow.com/questions/36814642/visual-studio-code-convert-spaces-to-tabs) on vscode? my tab size is 8 so looks weird on my vscode lol

Reply:
i dont understand lol video call pe dikha